cypress/e2e/listingDetails/completedLotteryListing.e2e.ts

```diff
@@ -0,0 +1,96 @@
+const listings = {
+  COMPLETED_LOTTERY: {
+    id: "a0W8H000000AmpKUAS",
+  },
+}
+
+const MOBILE_VIEWPORT_HEIGHT = 680
+const MOBILE_VIEWPORT_WIDTH = 420
+
+const visitListing = (mobile, language) => {
+  const langPart = language ? `/${language}` : ""
+  if (mobile) {
+    cy.viewport(MOBILE_VIEWPORT_WIDTH, MOBILE_VIEWPORT_HEIGHT)
+  }
+  cy.visit(`${langPart}/listings/${listings.COMPLETED_LOTTERY.id}?react=true`)
+}
+
+const clickLotteryResultsButton = (mobile) => {
+  if (mobile) {
+    cy.get(".detail-header__title").contains("Lottery").click()
+  }
+  cy.get("button").contains("View Lottery Results").click()
+}
+
+const searchForLotteryResults = () => {
+  cy.get("#lotterySearchNumber").type("01100972")
+  cy.get("form button svg").click()
+}
+
+const LOTTERY_BUTTON_TEXT = {
+  en: "View Lottery Results",
+  tl: "Ipakita ang mga Resulta ng Lottery",
+  zh: "查看抽籤結果",
+  es: "Ver los Resultados del Sorteo",
+}
+
+describe("Listing Details for Completed Lottery Listing", () => {
+  afterEach(() => {
+    // TODO: remove me once this is fixed. we shouldn't have to wait in between tests, but
+    // there is a rogue loading issue beyond the scope of this story
+    cy.wait(3000)
+  })
+
+  describe("Completed Lottery Rental Listing " + listings.COMPLETED_LOTTERY.id, () => {
+    it("displays in English", () => {
+      visitListing(true, "")
+      cy.contains(LOTTERY_BUTTON_TEXT.en)
+    })
```

Comment:
hmm, thinking we may not need to test multiple languages for every set of tests. worried about the entire test suite taking too long to run if we test all the languages for every feature we e2e test. do you think openListings.e2e.ts should give us enough confidence that human translations are working?